connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/PluginsTest.java

```diff
@@ -500,6 +500,11 @@ private void assertClassLoaderReadsVersionFromResource(
         );
         plugins = new Plugins(pluginProps, parent, new ClassLoaderFactory());
 
+        assertTrue("Should find plugin in plugin classloader",
+                plugins.converters().stream().anyMatch(desc -> desc.loader() instanceof PluginClassLoader));
+        assertTrue("Should find plugin in parent classloader",
+                plugins.converters().stream().anyMatch(desc -> !(desc.loader() instanceof PluginClassLoader)));
```

Comment:
Can we make this more explicit (especially since it would match the assertion message)?
```suggestion
                plugins.converters().stream().anyMatch(desc -> parent.equals(desc.loader())));
```